src/main.cc

```diff
@@ -222,6 +222,8 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
           // since reboot is required to apply/finalize the currently installed update (aka pending update)
           break;
         }
+      } else {
+        LOG_INFO << "A device is up-to-date";
```

Comment:
Can probably say either "Device is up-to-date" or "The device is up-to-date"

Reply:
ok